frame/contracts/src/storage/meter.rs

```diff
@@ -198,46 +286,39 @@ where
 	/// `info`: The info of the contract in question. `None` if the contract was terminated.
```

Comment:
```suggestion
	/// - `info`: The info of the contract in question. `None` if the contract was terminated.
```
just for proper doc formatting, this and 3 prev lines should have a list ticker (can't suggest the change for the other three as they're out of the PR changes scope :/), i.e.

```rust
	/// # Parameters
	///
	/// - `absorbed`: The child storage meter that should be absorbed.
	/// - `origin`: The origin that spawned the original root meter.
	/// - `contract`: The contract that this sub call belongs to.
	/// - `info`: The info of the contract in question. `None` if the contract was terminated.
```